osi_lane.proto

```diff
@@ -371,6 +371,14 @@ message Lane
         // \note \c Lane with \c #type = \c #TYPE_INTERSECTION use only free
         // lane boundaries.
         //
+        // \note Free boundaries should be defined in clockwise direction. The starting
```

Comment:
Is this only about the definition order of the boundary points or also about the order of referencing free lane boundaries in the list (e.g. [11, 14, 13, 12]?